docs/user/tutorials/01-53-expose-workload-extauth-client-credentials.md

```diff
@@ -0,0 +1,261 @@
+3. The client includes this JWT in the Authorization header when making requests to your secured Kyma workload.
+4. OAuth2 Proxy validates the JWT against SAP Cloud Identity Services and either allows the request to proceed to your workload or returns `401 Unauthorized` in response.
+
+Unlike the Authorization Code flow which requires browser redirects and user consent, the Client Credentials flow enables direct server-to-server communication using only the application's own credentials.
+
+For more information, see [OAuth 2.0 RFC 6749](https://datatracker.ietf.org/doc/html/rfc6749#section-1.3.4) and [Client Credentials Flow](https://auth0.com/docs/get-started/authentication-and-authorization-flow/client-credentials-flow#how-it-works).
+
+Follow these steps:
+1. [Create and Configure an OpenID Connect Application](#create-and-configure-an-openid-connect-application)
+2. [Get a JWT](#get-a-jwt)
+3. [Deploy OAuth2 Proxy as an External Authorizer](#deploy-oauth2-proxy-as-an-external-authorizer)
+4. [Expose Your Workload Using extAuth APIRule](#expose-your-workload-using-apirule-with-extauth)
+
+## Procedure
+
+### Create and Configure OpenID Connect Application
+You need an identity provider to issue JWTs. Creating an OpenID Connect application allows SAP Cloud Identity Services to act as your issuer and manage authentication for your workloads.
+
+1. Sign in to the administration console for SAP Cloud Identity Services. See [Access Admin Console](https://help.sap.com/docs/cloud-identity-services/cloud-identity-services/accessing-administration-console?locale=en-US&version=Cloud).
+
+2. Create an OpenID Connect Application.
+
+   1. Go to **Application Resources** > **Application**.
+   2. Choose **Create**, provide the application name, and select the OpenID Connect radio button. 
+      For more configuration options, see [Create OpenID Connect Application](https://help.sap.com/docs/cloud-identity-services/cloud-identity-services/create-openid-connect-application-299ae2f07a6646768cbc881c4d368dac?locale=en-US&version=Cloud).
+   3. Choose **+Create**.
+
+3. Configure OpenID Connect Application for the Client Credentials flow.
+   
+   1. In the **Trust > Single Sign-On** section of your created application, choose **OpenID Connect Configuration**.
+   2. Provide the name.
+   3. In the **Grant types** section, check **Client Credentials**.
+      For more configuration options, see [Configure OpenID Connect Application for Client Credentials Flow](https://help.sap.com/docs/cloud-identity-services/cloud-identity-services/client-cred-configure-openid-connect-application-for-client-credentials-flow?locale=en-US&version=Cloud).
+   4. Choose **Save**.
+
+4. Configure a secret for API authentication.
+
+   1. In the **Application API** section of your created application, choose **Client Authentication**.
+   2. In the **Secrets** section, choose **Add**.
+   3. Choose the OpenID API access and provide other configuration as needed.
```

Comment:
To make sure: no UI element is mentioned in this sentence, right?

Reply:
both "Open ID" and API access are UI elements. I added bold for "Open ID" and left API Access in regular font but I'm not sure if that's correct. WDYT?
<img width="474" height="609" alt="Screenshot 2025-12-03 at 06 43 04" src="https://github.com/user-attachments/assets/cd7eb445-1aa2-4c4a-b7f5-8ebfa802cc64" />